codalab/worker/bundle_manager.py

```diff
@@ -62,8 +63,8 @@ def parse(to_value, field):
         self._max_request_memory = parse(formatting.parse_size, 'max_request_memory')
         self._max_request_disk = parse(formatting.parse_size, 'max_request_disk')
 
-        self._default_request_cpus = config.get('default_request_cpus')
-        self._default_request_gpus = config.get('default_request_gpus')
+        self._default_request_cpus = config.get('default_request_cpus', 1)
```

Comment:
Why have defaults here for some fields but not others?

Reply:
actually i don't think these are used at all; none of our workers have these _default_* configs defined. I think I'll just revert it to whatever it was before.

Reply:
nevermind. These fields are used by _compute_request_*pus and they must be integers rather than NoneType otherwise they crash something else down the line.